### PR TITLE
Fix direction in doctest

### PR DESCRIPTION
**Pull request by himkt**
(related https://github.com/optuna/optuna/pull/1021#discussion_r393507789)
`clf.score` returns a mean accuracy of classification, so objective functions should be maximized.

- MLPClassifier: https://scikit-learn.org/stable/modules/generated/sklearn.neural_network.MLPClassifier.html#sklearn.neural_network.MLPClassifier.score
- SVC: https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html#sklearn.svm.SVC.score
- GradientBoostingClassifier: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html#sklearn.ensemble.GradientBoostingClassifier.score
- RandomForestClassifier: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html#sklearn.ensemble.RandomForestClassifier.score
- SGDClassifier: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html#sklearn.linear_model.SGDClassifier.score

### CODE REVIEW

**Comment by hvy** [comment]
Let me modify the title of this PR to match the format of our release notes.
